test/expect/TestScript.test_annot_ast_mypy_method.expect

```diff
@@ -1,9 +1,9 @@
-foo(ClassType<FooModule> self, Tensor x, (Tensor, Tensor) y) -> ((Tensor, Tensor))
-foo(ClassType<FooModule> self, Tensor x, (Tensor, Tensor) y) -> ((Tensor, Tensor))
-foo(ClassType<FooModule> self, str x, (Tensor, Tensor) y) -> ((str, str))
-foo(ClassType<FooModule> self, int x, (Tensor, Tensor) y) -> ((int, int))
-foo(ClassType<FooModule> self, bool x, (Tensor, Tensor) y) -> ((bool, bool))
-foo(ClassType<FooModule> self, float[3] x, (Tensor, Tensor) y) -> ((float[], float[]))
-foo(ClassType<FooModule> self, int[2] x, (Tensor, Tensor) y) -> ((int[], int[]))
-foo(ClassType<FooModule> self, int[] x, (Tensor, Tensor) y) -> ((int[], int[]))
-foo(ClassType<FooModule> self, int? x, (Tensor, Tensor) y) -> ((int?, int?))
\ No newline at end of file
+foo( self, Tensor x, (Tensor, Tensor) y) -> ((Tensor, Tensor))
```

Comment:
This looks wrong, the name is an empty string here?

Reply:
It was too hard to figure out how to decay the name to the point where textual comparison would match in the presence of name mangling. Ideally we would get rid of these expect files altogether